src/FixedFrequencyAgent.hpp

```diff
@@ -93,26 +94,10 @@ namespace geopm
             const PlatformTopo &m_platform_topo;
 
             geopm_time_s m_last_wait;
-            const double M_WAIT_SEC;
-
+            double M_WAIT_SEC;
             bool m_do_write_batch;
```

Comment:
If this is unused it should be removed.

Reply:
this is still being used, I needed it to not be a constant as it is set by the SAMPLE_PERIOD in the policy.